samples/sampler/stories/default/Picker.js

```diff
@@ -16,6 +16,7 @@ storiesOf('Agate', module)
 				<Picker
 					disabled={boolean('disabled', Config)}
 					onChange={action('onChange')}
+					orientation={select('orientation', ['vertical', 'horizontal'], Config)}
```

Comment:
The `Config` does not have the default value for the `orientation`. Please check the comment below for `RangePicker` sampler - https://github.com/enactjs/agate/pull/272/files#r444642557.